deploy/crds/toolchain_v1alpha1_masteruserrecord_cr.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: toolchain.dev.openshift.com/v1alpha1
 kind: MasterUserRecord
 metadata:
-  name: johnsmith #username
+  name: johnsmith # username
```

Comment:
Those are generated yamls. We should not change them manually.

Reply:
ah right. So let me exclude the YAML files in `deploy/crds`, then

Reply:
reverted in d20fe7f